docs/topics/develop/patterns/asynchronous-processing.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 6
+title: Asynchronous Processing
+sidebar_label: Asynchronous Processing
+description: Learn the how to use Topics and Webhooks to process data asynchronously
+keywords:
+ - topics
+ - webhooks
+ - eda
+ - momento
+ - event-driven architecture
+ - step functions
+---
+
+# Asynchronous Processing with Momento
+
+With Momento Topics, you can subscribe to messages on a Topic, as well as publish messages to a new Topic. Webhooks allow you to connect these Topics to stateless consumers, which can then process these events asynchronously. Whether this is aggregating events by `topic_id`, saving each event to a database, or using the payload to trigger a step function, webhooks give you the flexibility to process these events how you want to.
+
+![Architecture](@site/static/img/topics/patterns/asynchronous-processing.png)
+
+## Pre-requisites
+1. A public facing endpoint to receive webhook events. This endpoint must accept POST requests, and be able to be receive inbound calls from Momento. The structure of the POST body will be of the format
+```typescript
+{
+    cache: string;
+    topic: string;
+    event_timestamp: number;
+    publish_timestamp: number;
+    topic_sequence_number: number;
+    token_id: string; // is an empty string if the message was published with an api key that does not contain a token_id
+    text: string; // this is the message that was published
+}
+```
+
+## Getting Started
+1. [Create Momento Cache](https://console.gomomento.com/caches/create)
```

Comment:
"Create a cache in the Momento console."

The way it's worded now sounds like you're talking about the product name